### ISSUE TITLE
Migration to GitLab

### ISSUE THREAD

**Issue by supremesonicbrazil** [supremesonicbrazil/GBML#38]
Call me a tinfoil hatter, conspiracy theorist or whatever, I don't trust Microsoft even if they bake me a cake the size of Everest. I've seen the shit they did with Skype, Mojang, and even their own OS. I'm not convinced this time will be any different.

I may be tied right now due to issues I have to solve with v1.4.0.0 (which is already done I guess, only thing left now is basically learning to do deb packaging the right way), plus this semester in college still requires me to use GitHub (which should end in a few weeks from now, exams are coming). But after I've sorted out both of these things, I'll be migrating this repo to GitLab as well, just to be sure.

That doesn't mean I'm gonna straight up delete this, it's just a safety countermeasure (for now). I just want to have this program safe and sound somewhere else in case something happens, but I'll (probably, not sure if) still be updating this repo until God knows when. I'll keep watching closely what happens on both sides, and I'll let y'all know where this is headed for sure if the day comes.

You guys might already know about [this repo](https://github.com/upend/IF_MS_BUYS_GITHUB_IMMA_OUT), if not, well, there ya have it.

**Comment by maxbla** [comment]
I agree that a migration isn't a bad idea, as Microsoft's history with open source is not encouraging. If you move to Gitlab, make sure to include a link to the new reop in this thread as well as prominently displayed in the readme.

How are you planning on migrating issues? I was planning to submit a few here very soon.

**Comment by supremesonicbrazil** [comment]
GitLab has [a way to migrate repos](https://www.youtube.com/watch?v=VYOXuOg9tQI) already, at 0:25 there's a list of what gets imported in the process:

* Git Repository
  * Commit History
  * Branches
* Pull Requests
* **Issues** (there ya have it :D)
* Labels
* Comments on issues & PRs
* Wiki Pages
* Milestones

Unfortunately I think it doesn't seem to import stars, forks or release tags, but that's not too big of a problem I believe. I'll be sure to link to the new repo here as well. Regarding the issues, I'll try to work on them and properly close them first before migrating.

**Comment by supremesonicbrazil** [comment]
@maxbla pinging you here because I'm aware of your last opened issues - #42 , #43 and #44 .

I've tested the GitLab import just to make sure everything would go smooth, and it actually did. Pretty much everything was imported - your issues as well, under your name (although GitLab says I've opened them instead of you, but your name's there in "Created by: "). It really didn't copy the stars and forks (which is kinda obvious tbh), but surprisingly it did copy the release tags, though not with the deb/rpm binaries, only the source code. Probably I'll have to release it again there but that's a minor problem I guess.

I think I'm ready to move there today, not only with a fresh new platform to work on, but also because I want to address issues properly now, using milestones and labels the way it should be. I hope you don't mind, your issues (and any future issues that get here before I decide to close it all down) will be worked on there from now on. I'll link them properly after everything is set, then I'll start the "shutdown" process for this one.

Thanks for the collaboration, not only yours but from everyone else as well :)
